tools/dev/run_dev_server.sh

```diff
@@ -1161,6 +1161,20 @@ local-usage-s3)
     echo "Dashboards should be available at:"
     echo "- Overview: http://localhost:3000/d/weaviate-overview/weaviate-overview"
     ;;
+  local-contextualai)
+    echo "Starting Weaviate with Contextual AI modules..."
+    echo "Make sure to set CONTEXTUAL_API_KEY environment variable"
+    
+    AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED=true \
+    ENABLE_API_BASED_MODULES=true \
```

Comment:
Hey @antas-marcin - did we deprecate `ENABLE_API_BASED_MODULES` in favour of `API_BASED_MODULES_DISABLED`?